packages/app-builder/src/routes/ressources+/settings+/analytics+/filters.ts

```diff
@@ -0,0 +1,45 @@
+import { type ExportedFields } from '@app-builder/models/data-model';
+import { isAdmin } from '@app-builder/models/user';
+import { initServerServices } from '@app-builder/services/init.server';
+import { getRoute } from '@app-builder/utils/routes';
+import { type ActionFunctionArgs, redirect } from '@remix-run/node';
+
+export async function action({ request }: ActionFunctionArgs) {
+  const { authService } = initServerServices(request);
+  const { user, dataModelRepository } = await authService.isAuthenticated(request, {
+    failureRedirect: getRoute('/sign-in'),
+  });
+
+  if (!isAdmin(user)) {
+    return redirect(getRoute('/'));
+  }
```

Comment:
I don't think we need this. And furthermore, if we catch the redirect, the frontend will redirect to the '/' without any notice.